### PR TITLE
Weather Api

### LINKED ISSUE #33: Fetch Weather Api and Render It

### PR DESCRIPTION
**Pull request by JayDampitan**
## Changes
1. Added weatherApi.ts in Api folder.
2. Fetch weather Api using typescript. 
3. Render current location, temperature, time and date in weather component. 

## Purpose
Enable users to see the current locations temperature, time and date in the web app. 

Closes #33

### CODE REVIEW

File: the-toad-tribune/src/layout/Weather.tsx [hunk 1]
```diff
@@ -1,7 +1,17 @@
 import styled from "styled-components";
+import { WeatherProp } from "../api/weatherApi";
 
-const Weather = () => {
-  return <WeatherStyles>Weather</WeatherStyles>;
+const Weather: React.FC<WeatherProp> = ({ weatherResponse }) => {
+  console.log(weatherResponse)
```
Comment (connielion) [inline the-toad-tribune/src/layout/Weather.tsx:5]: Do you need this still?

**Review by connielion — APPROVED** [review]
Looks good overall.

**Review by Gjhnsn — APPROVED** [review]
Lots of weather data! Looks good!

File: the-toad-tribune/src/layout/Weather.tsx (hunk 1 above)
Comment (JayDampitan) [reply the-toad-tribune/src/layout/Weather.tsx:5]: Good catch Connie!

**Pull request merged**